.github/workflows/release.yml

```diff
@@ -30,10 +30,14 @@ jobs:
       - run: npm run build --workspaces
       # run tests just in case
       - run: npm test
+      # update versions and dependencies using tag
+      - run: npm run version -- ${GITHUB_REF#refs/tags/v}
+      # commit changes to main branch
+      - run: git add .; git commit -m "Releases ${GITHUB_REF#refs/tags/v}"; git push origin HEAD:main;
```

Comment:
The "git push origin HEAD:main" is the one thing I'm not sure will work, given our repo settings